.gitlab-ci-full-integration.yml

```diff
@@ -0,0 +1,48 @@
+test:integration:
```

Comment:
The current pipeline takes more than 2 hours. The lowest hanging fruit to speed it up is to separate this into two jobs: open source tests and enterprise tests. This is the strategy on the current pipeline where it takes 1.5h + 1h (in parallel, so 1.5h human time) and I believe this one will run faster here.

Reply:
I refactored the compatibility tests which amounts to a total of 1h execution time. Gonna give it another spin :wheel: